index.js

```diff
@@ -166,8 +167,13 @@ async function checkHeartbeat() {
   }
 }
 
-async function handleSensorRequest(currentLocationId) {
-  const statemachine = new StateMachine(currentLocationId)
+async function handleSensorRequest(currentLocationId, radarType) {
+  let statemachine
+  if (radarType === 'XeThru') {
```

Comment:
Can you please make `XeThru` and `Innosent` into an Enum and use it wherever you have these strings hardcoded?